cmd/registry/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ev
+
+go install github.com/google/go-containerregistry/cmd/registry
+go install github.com/google/go-containerregistry/cmd/crane
```

Comment:
Can this be `go build ./cmd/crane -o crane`, to make it clear that it's building the local source, and not pulling and installing code from the remote GitHub repo?

Reply:
Done.